src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -207,9 +207,9 @@ static AnimationInfo sAnimationInfo[] = {
 
 typedef struct {
     /* 0x00 */ u8 headInfoIndex;  // EnHyHeadIndex
-    /* 0x01 */ u8 skelInfoIndex2; // EnHySkeletonIndex, see EnHy#objBankIndexSkel2
+    /* 0x01 */ u8 skelInfoIndex2; // EnHySkeletonIndex, see EnHy.objBankIndexSkel2
     /* 0x02 */ Color_RGBA8 envColorSeg8;
-    /* 0x06 */ u8 skelInfoIndex1; // EnHySkeletonIndex, see EnHy#objBankIndexSkel1
+    /* 0x06 */ u8 skelInfoIndex1; // EnHySkeletonIndex, see EnHy.objBankIndexSkel1
```

Comment:
since this is within the actor itself, is it even necessary to say `EnHy.thing`? would just mention the struct member by itself personally

Reply:
the comment is in `EnHyModelInfo` in `z_en_hy.c`, `objBankIndexSkel1` is in `EnHy` in `z_en_hy.h`

One could figure out where `objBankIndexSkel1` is from a quick ctrl+f in the file, but imo we may as well be explicit here with the crazy amount of structs